### ISSUE TITLE
Bar data.onclick is not handled

### ISSUE THREAD

**Issue by creage** [naver/billboard.js#1664]
## Bar `data.onclick` is not handled anymore
I have a code that should be executed when user clicks on a particular bar, and it was working on v1, but not in v2 anymore.

```js
{
...
data:{
    type: bar(),
    onclick: function(){ alert(1); }
}
...
```

Clicking on bars does nothing.

**Comment by netil** [comment]
The issue was already solved #1619 
Can you try by installing the `next` tagged version?

```js
npm i billboard.js@next
```

**Comment by creage** [comment]
Fix confirmed, works fine on latest beta. Sorry for not checking it before creating the issue.
